src/runtime/pay-flow.js

```diff
@@ -71,21 +80,37 @@ export class PayStartFlow {
    * @return {!Promise}
    */
   start() {
-    // Start/cancel events.
-    this.deps_.callbacks().triggerFlowStarted(SubscriptionFlows.SUBSCRIBE, {
+    const triggerFlowKeys = {
       'sku': this.sku_,
-    });
+    };
+
+    const swgKeys = {
```

Comment:
I'd call it `swgPayload` or `swgPaymentRequest`

Reply:
done